tests/cxx/final_class_impl.h

```diff
@@ -0,0 +1,13 @@
+//===--- final_class_impl.h - test input file for iwyu --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+template <typename T>
+class A final {};
```

Comment:
Could you also add a non-template class with `final`? It seems to work already, I just want to make sure it's covered.

If you name `A` as `FinalTemplate` instead, you can name the new class `FinalClass` and it's more obvious what their role is.